packages/overlay/imperative-api.md

```diff
@@ -84,6 +84,135 @@ Some Overlays will always be passed focus (e.g. modal or page Overlays). When th
 
 The `trigger` option accepts an `HTMLElement` or a `VirtualTrigger` from which to position the Overlay.
 
+-   You can import the `VirtualTrigger` class from the overlay package to create a virtual trigger that can be used to position an Overlay. This is useful when you want to position an Overlay relative to a point on the screen that is not an element in the DOM, like the mouse cursor.
+
+### Using a virtual trigger
```

Comment:
A second or togglable version of this that showed the way that https://github.com/adobe/spectrum-web-components/issues/2568#issuecomment-1879457929 could be addressed might be nice here as well.

Reply:
Should I do with an virtual trigger example or a separate simpler isolated example